### PR TITLE
Add notebook on how to create a Service

### PR DESCRIPTION
**Pull request by djkonro**

### CODE REVIEW

File: examples/notebooks/create_service.ipynb [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "spec = client.V1ServiceSpec()\n",
+    "spec.selector = {'app': 'MyApp'}\n",
+    "spec.ports = [client.V1ServicePort(protocol='TCP', port=80, target_port=9376)]\n",
```
Comment (djkonro) [inline examples/notebooks/create_service.ipynb]: @mbohlool Please, why does the documentation (https://github.com/kubernetes-incubator/client-python/blob/master/kubernetes/docs/V1ServicePort.md) say `target_port` is of type `str`, but when I do `target_port='9376'` I get the errors below.

`kubernetes.client.rest.ApiException: (422)`
`Reason: Unprocessable Entity`
`HTTP response headers: HTTPHeaderDict({'Date': 'Thu, 08 Jun 2017 22:42:31 GMT', 'Content-Length': '468', 'Content-Type': 'application/json'})`
`HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"Service \"my-service\" is invalid:` `spec.ports[0].targetPort: Invalid value: \"9376\": must contain at least one letter or number (a-z, 0-9)","reason":"Invalid","details":{"name":"my-service","kind":"Service","causes":[{"reason":"FieldValueInvalid","message":"Invalid value: \"9376\": must contain at least one letter or number (a-z, 0-9)","field":"spec.ports[0].targetPort"}]},"code":422}`

Comment (mbohlool) [inline examples/notebooks/create_service.ipynb]: Ignore docs and pass a number instead of string. Let me know if it worked and I will explain why.

Comment (djkonro) [inline examples/notebooks/create_service.ipynb]: Thanks,  passing a number works fine.

File: examples/notebooks/create_service.ipynb [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "How to create a Service\n",
+    "=============\n",
+    "\n",
+    "In this notebook, we show you how to create a [Service](https://kubernetes.io/docs/concepts/services-networking/service/). \n",
+    "A Service in Kubernetes is a REST object, similar to a Pod. It is an abstraction which defines a logical set of Pods and a policy by which to access them - sometimes called a micro-service."
```
Comment (sebgoa) [inline examples/notebooks/create_service.ipynb]: A service is a key Kubernetes API resource. It defines a networking abstraction to route traffic to a particular set of Pods using a label selection.

File: examples/notebooks/create_service.ipynb [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "### Provide Service .spec description\n",
+    "Set Service object named **my-service** to target TCP port **9376** on any Pod with the **app=MyApp** label."
```
Comment (sebgoa) [inline examples/notebooks/create_service.ipynb]: ..the label selection allows Kubernetes to determine which Pod should receive traffic when the service is used.

**Comment by sebgoa** [comment]
/lgtm

@mbohlool

**Pull request merged**